development.ini.template

```diff
@@ -7,7 +7,7 @@
 use = config:base.ini#app
 sqlalchemy.url = postgresql://postgres@localhost:5441/postgres?host=/tmp/snovault/pgdata
 blob_bucket = cgap-unit-testing-blobs
-metadata_bundles_bucket = cgap-unit-testing-metadata-bundles
+metadata_bundles_bucket = smaht-production-application-metadata-bundles
```

Comment:
This is a dev ini file but it's supposed to use the production metadata bundles bucket? That's not necessarily horrible, but I think the reason for wanting the analog of the cgap-unit-testing-metadata-bundles is so we can clean it from time to time and so people who lack production credentials can browse test data to debug it. @willronchetti, thoughts?

Reply:
I think an alternative bucket should be created yes similar to the devtest buckets

Reply:
Created bucket: smaht-unit-testing-metadata-bundles